public/modules/dashboards/data/static_en.json

```diff
@@ -12,6 +12,7 @@
   "share_url": "Share full current settings (URL)",
   "export_csv": "Export (CSV)",
   "export_geojson": "Export (GEOJSON)",
+  "signup_mailing": "Sign up to mailing list",
```

Comment:
This doesn't include the "Export (PDF)" or "Join WhatsApp group" values; shouldn't all strings align between these 2 files? Or can we remove 1 of the 2?